emcc.py

```diff
@@ -1876,8 +1874,8 @@ def phase_linker_setup(options, state, newargs, user_settings):
         # Include dynCall() function by default in DYNCALLS builds in classic runtime; in MINIMAL_RUNTIME, must add this explicitly.
         settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$dynCall']
 
-      if settings.ASSERTIONS and not settings.EXIT_RUNTIME:
-        # "checkUnflushedContent()" depends on warnOnce
+      if settings.ASSERTIONS:
```

Comment:
Why is there an `EXIT_RUNTIME` change in this PR?

Reply:
In the old code we were including `warnOnce` only to support checkUnflushedContent which happens on exit (only when EXIT_RUNTIME is defined).

Actually `warnOnce` is also needed for *all* debug builds wince its used in `missingLibrarySymbol()` which is unconditionally included when ASSERTIONS are enabled.   We didn't notice this dependency before because of LEGACY_RUNTIME being enabled by default.